openlibrary/plugins/openlibrary/js/add_new_field.js

```diff
@@ -42,7 +42,7 @@ export default function($){
             });
 
             // handle cancel
-            $(options.href).bind('cbox_closed', function() {
+            $(options.href).on('cbox_closed', function() {
```

Comment:
Consider creating this once to $href since it's created later as well.

Reply:
`45:13  error  '$href' is not defined  no-undef`
I get this error

Reply:
What i mean is
```
const $href = $(options.href;
$href.on('cbox_closed', ...
...
$('form:first', $href).on('submit...
```

Reply:
Done.